test/extended/builds/secrets.go

```diff
@@ -40,6 +40,7 @@ var _ = g.Describe("[Feature:Builds][Slow] can use build secrets", func() {
 		g.Describe("build with secrets", func() {
 
 			g.It("should contain secrets during the source strategy build", func() {
+				g.Skip("skipping on 3.10 because the 3.10 AMIs have the bad docker version")
```

Comment:
@gabemontero sounds like we need openshift-specific BZs for the docker patch in RHEL 7?

Reply:
yeah @bparees and I discussed this in a devex slack thread a couple of days ago ... I'll clone the existing rhel7 bz and put it once the RHEL bz ships 

@bparees did not want to burden QE with going after pre-release RHEL patches